config.toml

```diff
@@ -214,35 +214,24 @@ no = 'Sorry to hear that. Please <a href="https://github.com/kubeflow/website/is
 [params.links]
 # Docsy: "User" links will show on left side of footer and in the community page if you have one.
 [[params.links.user]]
-	name ="Twitter"
+	name = "Twitter"
 	url = "https://twitter.com/kubeflow"
 	icon = "fab fa-twitter"
-        desc = "Follow us on Twitter to get the latest news"
+    desc = "Follow us on Twitter to get the latest news!"
 [[params.links.user]]
-	name = "GitHub"
-	url = "https://github.com/kubeflow/kubeflow"
-	icon = "fab fa-github"
-        desc = "Development takes place here!"
-[[params.links.user]]
-	name = "Slack"
-	url = "https://kubeflow.slack.com/join/shared_invite/zt-cpr020z4-PfcAue_2nw67~iIDy7maAQ"
+	name = "Slack Server"
+	url = "/docs/about/kubeflow-slack-workspace"
 	icon = "fab fa-slack"
-        desc = "Chat with other project contributors"
+    desc = "Join the Kubeflow Slack Workspace!"
 [[params.links.user]]
 	name = "Mailing list"
-	url = "https://groups.google.com/forum/#!forum/kubeflow-discuss"
+	url = "/docs/about/kubeflow-mailing-list"
```

Comment:
I guess the hyper link here is one of the reasons for breaking up sections in `Community` page to different pages.

But I would suggest to use hashtag instead: `/docs/about/community/#mailing-lists` in here, which makes it easier for people to see all community contact approach if they navigate from this footer icon.

Reply:
I have changed it back to a single community page.